Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/CubeControls.java

```diff
@@ -35,5 +39,18 @@ public void initDefaultCommand() {
         // Set the default command for a subsystem here.
         //setDefaultCommand(new MySpecialCommand());
     }
+    public void intake(){
```

Comment:
`Robot.cubeControls.*IntakeMotor` is not needed in these methods. Because we are already in the `CubeControls` subsystem, we have access to the `climbMotor`. So you can drop `Robot.cubeControls.` and just use `leftIntakeMotor` or `rightIntakeMotor`.